projects/frontend/data-pipelines/gui/e2e/fixtures/lib/explore/additional-test-job.json

```diff
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "contacts": {
+      "notified_on_job_deploy": [
+        "auserov3@vmware.com"
+      ],
+      "notified_on_job_failure_platform_error": [
+        "buserov3@vmware.com"
+      ],
+      "notified_on_job_failure_user_error": [
+        "cuserov3@vmware.com"
+      ],
+      "notified_on_job_success": [
+        "duserov3@vmware.com"
+      ]
+    },
+    "db_default_type": "IMPALA",
+    "generate_keytab": false,
+    "schedule": {
+      "schedule_cron": "0 0 03 * 5"
+    }
+  },
+  "description": "Test description 3 with a long text on the line",
+  "job_name": "e2e-cp-$env-placeholder$-a-test-job-3",
```

Comment:
'cp' stands for Control Plane. 

May be it is more natural to change it with VDK or something along these lines.

Reply:
That's a good point; my suggestion here is to keep the naming as-is, and have the changes made during the CI/CD e2e introduction, where they run and could be tested and verified 😊

Reply:
sure. Agree.